location_vs_expression_relationships/1_genome_dist_vs_expression_dist.py

```diff
@@ -59,23 +59,24 @@
 )
 # -
 
-# ### Import module memberships
+# ### Import gene ids
```

Comment:
Are the two functions in this notebook doing similar things? If so could it be collapsed into one function?

Reply:
So they're both finding relationships but in different spaces - one is genome space (so who is your nearest neighbor, core or acc?) and the other is in expression space (who are you most co-expressed with, core or acc?). I think conceptually they are similar but the way I need to calculate things is a little different. Since the functions are already fairly large I'd rather keep them separate, mainly for my own understanding I think having them separate helps. But good question!